modules/rest/src/main/java/eu/cloudnetservice/modules/rest/v2/V2HttpHandlerQRCode.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019-2022 CloudNetService team & contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package eu.cloudnetservice.modules.rest.v2;
+
+import eu.cloudnetservice.driver.CloudNetDriver;
+import eu.cloudnetservice.driver.network.http.HttpContext;
+import eu.cloudnetservice.driver.network.http.HttpRequest;
+import eu.cloudnetservice.driver.network.http.HttpResponseCode;
+import eu.cloudnetservice.driver.permission.PermissionUser;
+import eu.cloudnetservice.modules.rest.RestUtil;
+import eu.cloudnetservice.modules.rest.util.QRCodeBuilder;
+import eu.cloudnetservice.node.http.V2HttpHandler;
+import java.io.ByteArrayOutputStream;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import org.jetbrains.annotations.NotNull;
+
+public class V2HttpHandlerQRCode extends V2HttpHandler {
+  private final ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+
+  public V2HttpHandlerQRCode() {
+    super(null, "GET");
```

Comment:
Why dont we need a permission here?

Reply:
This is just a easy way to deliver information. If i set a permission, you need also a auth head. Is not her required